### PR TITLE
Updated scrapper.py

### PR DESCRIPTION
**Pull request by abhay44**
As `number_of_pages` can be float, changed it to `int(number_of_changes)` so that it loops in the number of pages received.

### CODE REVIEW

File: ghscrap/scrapper.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def search_location(location, keyword=None, max_users=10, params={}):
     number_of_pages = ceil(number_of_users / 100)
     
     users = _get_users(response.json()['items'])
-    for i in range(2, number_of_pages):
+    for i in range(2, int(number_of_pages)):
```
Comment (kshitij10496) [inline ghscrap/scrapper.py]: @abhay44 I would prefer you type-cast the identifier `number_of_pages` at its definition itself ( L100).

Comment (abhay44) [reply ghscrap/scrapper.py]: @kshitij10496 Updated!

**Comment by kshitij10496** [comment]
@abhay44 Can you squash the commits into a single one please ?

**Comment by abhay44** [comment]
@kshitij10496 Done!

**Comment by kshitij10496** [comment]
@abhay44 Thanks for your patience.

**Pull request merged**